sites/NOAA_GFDL/doc/NOAA_GFDL.rst

```diff
@@ -69,7 +81,10 @@ This data source searches for model data produced using GFDL's in-house postproc
 
 <*CASE_ROOT_DIR*> should be set to the root of the postprocessing directory hierarchy (i.e., should end in ``/pp``).
 
---any-components    If this flag is set, the data source will return data from different model ``<component>``\s requested by the same POD. This is necessary for, e.g., PODs that compare data from different modeling realms. The default behavior is to require all variables requested by a POD to come from the same model ``<component>``.
+--component    If set, only run the package on data from the specified model component name. If this flag is *not* set, the data source will return data from different model ``<component>``\s requested by the same POD; see the description of the heuristics used for ``<component>`` selection below. This is necessary for, e.g., PODs that compare data from different modeling realms. 
+--chunk_freq    If set, only run the package on data with the specified timeseries chunk length. If not set, default behavior is to use the smallest chunks available.
```

Comment:
This is also good.  Is it redundant though since calls to analysis scripts can be tied to specific chunk lengths in the XML?  No need to change anything, just wondering if the behaviors are compatible and can co-exist.

Reply:
This is correct: these flags are there more for the use of the FRE wrapper script than the user's. The default behavior for the wrapper script is to ignore this information and consider all `component`s and `chunk_freq`s, but if the user passes `--component_only` to the wrapper script, the wrapper will get these attributes from FRE and pass them to the package via these flags.

4795b85de3a2a82dccdcad99783722ac10fecc4e adds a bit to the docs to clarify this.